pvlib/modelchain.py

```diff
@@ -926,7 +929,7 @@ def run_model(self, weather, times=None):
 
         Assigns attributes: solar_position, airmass, irradiance,
         total_irrad, effective_irradiance, weather, cell_temperature, aoi,
-        aoi_modifier, spectral_modifier, dc, ac, losses.
+        aoi_modifier, spectral_modifier, dc, ac, losses, diode_params.
```

Comment:
perhaps ", diode_params (if dc_model is a single diode model)" 

or in a new list:

Optionally assigns attributions: diode_params if dc_model is a single diode model.